tasks/eva_3399/src/main/groovy/eva3399/RemediationQC.groovy

```diff
@@ -2,6 +2,7 @@ package eva3399
 
```

Comment:
I will suggest reviewing the full file because I messed up and failed to open this up for review the last time before I merged